tool-plugins/intellij/src/main/java/io/ballerina/plugins/idea/debugger/BallerinaStackFrame.java

```diff
@@ -100,8 +108,27 @@ private VirtualFile findFile() {
         }
     }
 
+    private File searchFile(File file, String search) {
+        if (file.isDirectory()) {
+            File[] files = file.listFiles();
+            if(files!=null) {
```

Comment:
Shall we reduce the number of nested if statements here?

Reply:
Fixed